kibana-reports/public/components/main/reports_table.tsx

```diff
@@ -14,6 +14,7 @@
  */
 
 import React, { Fragment, useState } from 'react';
+import ReactDOM from "react-dom";
```

Comment:
why is reactdom needed?

Reply:
It's not, will remove- added it as a try while debugging